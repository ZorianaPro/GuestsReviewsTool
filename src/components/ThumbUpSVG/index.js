import React from 'react';

const ThumbUpSVG = (props) => (
  <svg className="ThumbUpSVG" {...props}>
    <use xlinkHref="#ThumbUpSymbol" />
  </svg>
);

export default ThumbUpSVG;
