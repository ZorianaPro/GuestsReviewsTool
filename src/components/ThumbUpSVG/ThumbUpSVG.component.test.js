import React from 'react';
import { shallow } from 'enzyme';
import ThumbUpSVG from './';

describe('ThumbUpSVG', () => {
  it('renders without crashing', () => {
    expect(shallow.bind(shallow, <ThumbUpSVG />)).not.toThrow();
  });

  it('renders as expected', () => {
    expect(shallow(<ThumbUpSVG />)).toMatchSnapshot();
  });
});
