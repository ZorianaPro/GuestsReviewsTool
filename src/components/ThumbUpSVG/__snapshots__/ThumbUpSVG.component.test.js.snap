// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThumbUpSVG renders as expected 1`] = `
<svg
  className="ThumbDownSVG"
>
  <use
    xlinkHref="#ThumbDownSymbol"
  />
</svg>
`;
